asf_welsh_energy_consultation/pipeline/process_data.py

```diff
@@ -384,3 +384,122 @@ def generate_age_data(wales_df):
     age_data["cumul_prop"] = age_data["percentage"].cumsum()
 
     return age_data
+
+
+def get_installations_per_year():
+    """
+    Get MCS installations per year for Wales.
+    Returns:
+        pandas.DataFrame of MCS installations per year in Wales.
+
+    """
+    mcs = get_enhanced_mcs()
+    mcs["n"] = 1
+    mcs["year"] = pd.to_datetime(mcs["commission_date"]).dt.year
+    installations_by_year = mcs.groupby("year")["n"].sum().reset_index()
```

Comment:
You can also do the following instead instead of reset_index() every time. No need to change it, just saying in case you didn't know :)
```suggestion
    installations_by_year = mcs.groupby("year", as_index=False)["n"].sum()
```

Reply:
Thanks! I didn't know :)